src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -501,6 +506,12 @@ private Task<bool> IsStartupSuccessfulAsync(ConnectionInfo sshConnectionInfo)
                 });
         }
 
+        private async Task<bool> MountWarningsExistAsync(ConnectionInfo sshConnectionInfo)
+        {
+            var warningCount = int.Parse((await ExecuteCommandOnVirtualMachineAsync(sshConnectionInfo, $"grep -c 'WARNING' {CromwellAzureRootDir}/mount.blobfuse.log || :")).Output);
```

Comment:
If grep doesn't find any WARNINGs in the file, grep -c returns 0, but the exit code will be non-zero, so using || : to ignore the error, but still getting the output from grep. -c means count the lines containing the string.